pkg/controller/installation/installation_controller.go

```diff
@@ -435,13 +445,28 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		installation.Status.Stage = integreatlyv1alpha1.StageName("complete")
 		metrics.RHMIStatusAvailable.Set(1)
 		retryRequeue.RequeueAfter = 5 * time.Minute
+		if installation.Spec.RebalancePods {
+			r.reconcilePodDistribution(installation)
+		}
 	}
 	metrics.SetRHMIStatus(installation)
 
 	err = r.updateStatusAndObject(originalInstallation, installation)
 	return retryRequeue, err
 }
 
+func (r *ReconcileInstallation) reconcilePodDistribution(installation *integreatlyv1alpha1.RHMI) {
+
+	serverClient, _ := k8sclient.New(r.restConfig, k8sclient.Options{})
```

Comment:
why are we ignoring this error? Should it be handled?

Reply:
I'm not sure if we should do anything beyond logging the error. The pod distribution functionality is an attempt to optimize the distribution but if it fails the operator can continue to work as normal. Failed distribution will then be highlighted to SRE through the PodDistributionAlert. wdyt?